README.md

```diff
@@ -1 +1,81 @@
-# angular-parsers-formatters-readme
\ No newline at end of file
+# $parsers and $formatters
+
+## Overview
+
+Parsers and formatters allow us to have different values in our view and our model for the same item. Don't be confused, they're really simple!
+
+## Objectives
+
+- Describe $parsers and $formatters
+- Use a $parser to change Model to View data
+- Use a $formatter to change View to Model data
+
+## $parsers and $formatters
+
+We can add parsers and formatters by requiring in the `ngModel` controller in our directive.
+
+```js
+function changeCase() {
```

Comment:
this directive almost looks like a custom filter? Will it simply format the text and input it into the DOM?